### PR TITLE
virtcontainers: Add to utils unit tests

### LINKED ISSUE #490: Expand unit tests for virtcontainers/utils/utils.go
There is about 55% coverage for virtcontainers/utils/utils.go, they can be expanded by covering additional functions. 

1.x issue is [#2877](https://github.com/kata-containers/runtime/issues/2877)

### PR DESCRIPTION
**Pull request by cmaf**
Expand unit tests for virtcontainers/utils/utils.go to include testing
CleanupFds, CPU calculations, ID string creation, and memory alignment
functions.

Fixes #490 

Signed-off-by: Chelsea Mafrica chelsea.e.mafrica@intel.com

Ported from 1.x [#2881](https://github.com/kata-containers/runtime/pull/2881)

### CODE REVIEW

**Comment by cmaf** [comment]
@chavafg The test for this PR should show whether or not kata-containers/kata-containers was successfully added to codecov

**Comment by egernst** [comment]
Good fix.  holding off on pressing the green button for now, as AFAICT codecov didn't run, and I assume this is something @chavafg and @cmaf want to look at?

**Comment by chavafg** [comment]
@cmaf can you add a line with `Depends-on: github.com/kata-containers/tests#2782` to the commit message and rerun the CI? That should take your changes on tests repo, run the unit tests and report the results.

**Comment by cmaf** [comment]
This PR is a test for [#2782](https://github.com/kata-containers/tests/pull/2782)

**Comment by chavafg** [comment]
Any idea of this issue? : 
This PR contains 4 commits, but seems that it is only looking for the `Fixes` in  the last 3 commits. The `Fixes` is in the first commit, so it is failing on the check. 
```
Run tim-actions/commit-message-checker-with-regex@v0.3.0
##[error]

  2a7e8f3f9c9bf55cc5e92355dd9dbd87edcb3abc    No "Fixes" found
  dc61b09e105b99bce38d2abeb026a7a201a81f94    No "Fixes" found
  8e2f51910e94c39723d71b455a1829efdb9e1e64    No "Fixes" found
```
Link to the run: https://github.com/kata-containers/kata-containers/pull/504/checks?check_run_id=981915514
/cc @amshinde @jodh-intel

**Comment by cmaf** [comment]
I removed the 2 commits trying to enable codecov reporting and saved them in another branch. They can be tested with a different unit test, this can be merged if it passes. 

/test-ubuntu

**Review by bergwolf — APPROVED** [review]
Thanks!

**Pull request merged**
